src/components/Notification/LoadingAnimation.jsx

```diff
@@ -33,7 +33,15 @@ const LoadingAnimation = ({ loadingItem, children }) => (
       padding: '20px'
     }}
   >
-    <Paper elevation={2} sx={{ display: 'inline-block', mx: '2px', padding: '20px' }}>
+    <Paper
+      elevation={2}
+      sx={{
+        display: 'inline-block',
+        mx: '2px',
+        padding: '20px',
+        backgroundColor: 'background.tint'
+      }}
+    >
```

Comment:
While this works for this animation, it consequently affects other uses of this animation as well, such as when loading contacts or documents.

<img width="796" alt="2024-07-01 (1)" src="https://github.com/codeforpdx/PASS/assets/83156697/5b783a3d-a7e1-4c4d-afb2-aaf5eebbf74a">

The box is now gray rather than white. Not a huge deal or anything, just an unintended side effect.

Best practice would be to carve out an exception for this. However, I'd like to do #643, which would refactor animations a tad anyway.